web/modules/custom/features/features.module

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @file
+ * Contains features.module.
+ */
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function features_form_user_register_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
+  if ($form_id == 'user_register_form') {
+    $form['account']['mail']['#description'] = NULL;
+
+    $form['account']['name']['#title'] = t('Login');
+    $form['account']['name']['#description'] = NULL;
+
+    // Set an additional validate function.
+    $form['#validate'][] = 'user_register_form_validate';
+  }
+}
+
+/**
+ * Implements user_register_form_validate().
+ */
+function user_register_form_validate(&$form, &$form_state) {
```

Comment:
You have to use the conventional names as wide as possible to avoid intersecting with existing functions already. 
The best callback's name is (notice please underscore first) _YOURMODULENAME_WHATISDOING() e.g. in this case:
function _features_user_register_form_validate(&$form, &$form_state)